docs/.sphinx/requirements.txt

```diff
@@ -31,7 +31,7 @@ click==8.1.3
     #   sphinx-external-toc
 click-log==0.4.0
     # via doxysphinx
-cryptography==41.0.3
+cryptography==41.0.4
```

Comment:
@fiona-gladwin Why is this file getting changed?

Reply:
Fixed after merge